stubs/pynput/pynput/_util.pyi

```diff
@@ -29,7 +29,7 @@ class AbstractListener(threading.Thread):
     _thread: threading.Thread  # undocumented
     _condition: threading.Condition  # undocumented
     _ready: bool  # undocumented
-    _queue: Queue[sys._OptExcInfo | None]  # undocumented
+    _queue: Queue[sys._OptExcInfo | None]  # undocumented  # TODO: use _typeshed.OptExcInfo after release of mypy 0.950
```

Comment:
I believe the release branch of 0.950 has already been cut, so this comment may or may not be accurate